config/initializers/commontator.rb

```diff
@@ -1,187 +1,249 @@
-
-  # Proc called with thread and user as arguments
-  # Returns true iif the user is a moderator for that particular thread
-  # and can delete users' comments in the thread or close it
-  # Default: lambda { |thread, user| false } (no thread-specific moderators)
-  config.can_edit_thread_proc = lambda { |thread, user| false }
-
-  # Proc called with the commontable object as argument
-  # Returns the name by which the commontable object will be called in email messages
-  # If you have multiple commontable models, you may want to pass this
-  # configuration value as an argument to acts_as_commontable in each one
-  # Default: lambda { |commontable| "#{commontable.class.name} ##{commontable.id}" }
-  config.commontable_name_proc = lambda { |commontable| "#{ commontable.class.name } ##{ commontable.id }" }
-
-  # Proc called with main_app and commontable objects as arguments
-  # Return the url that contains the commontable's thread to be used in the subscription email
-  # The application's routes can be accessed using the main_app object
-  # Default: lambda { |main_app, commontable| main_app.polymorphic_url(commontable) }
+  # Notes:
+  #   :e is usually used in forums (discussions)
+  #   :l is usually used in blogs (opinions)
+  #   :ve and :vl are usually used where it makes sense to rate comments
+  #     based on usefulness (q&a, reviews, guides, etc.)
+  # If :l is selected, the "reply to thread" form will appear before the comments
+  # Otherwise, it will appear after the comments
+  # Default: :e
+  config.comment_order = :e
+
+  # new_comment_style
+  # Type: Symbol
+  # How to display the "new comment" form
+  # Valid options:
+  #   :t (always present in the thread's page)
+  #   :l (link to the form; opens in the same page using JS)
+  # Not yet implemented:
+  #   :n (link to the form; opens in a new window)
+  # Default: :l
+  config.new_comment_style = :l
+
+  # comments_per_page
+  # Type: Fixnum or nil
+  # Number of comments to display in each page
+  # Set to nil to disable pagination
+  # Any other value requires the will_paginate gem
+  # Default: nil (no pagination)
+  config.comments_per_page = nil
+
+  # thread_subscription
+  # Type: Symbol
+  # Whether users can subscribe to threads to receive activity email notifications
+  # Valid options:
+  #   :n (no subscriptions)
+  #   :a (automatically subscribe when you comment; cannot do it manually)
+  #   :m (manual subscriptions only)
+  #   :b (both automatic, when commenting, and manual)
+  # Default: :n
+  config.thread_subscription = :n
+
+  # email_from_proc
+  # Type: Proc
+  # Arguments: a thread (Commontator::Thread)
+  # Returns: the address emails are sent "from" (String)
+  # Important: If using subscriptions, change this to at least match your domain name
+  # Default: lambda { |thread|
+  #                   "no-reply@#{Rails.application.class.parent.to_s.downcase}.com" }
+  config.email_from_proc = lambda { |thread|
```

Comment:
Unused block argument - `thread`. If it's necessary, use `_` or `_thread` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.